github-issues.go

```diff
@@ -44,104 +39,66 @@ type GitIssues struct {
 	Repo      string `json:"repository_url"`
 }
 
-// RepoIssues - holds all issues per repo.
-type RepoIssues struct {
-	URL           string `json:"url"`
-	RepositoryURL string `json:"repository_url"`
-	HTMLURL       string `json:"html_url"`
-	ID            int    `json:"id"`
-	Number        int    `json:"number"`
-	Title         string `json:"title"`
-	User          struct {
-		Login string `json:"login"`
-	} `json:"user"`
-	Labels []struct {
-		ID      int    `json:"id"`
-		URL     string `json:"url"`
-		Name    string `json:"name"`
-		Color   string `json:"color"`
-		Default bool   `json:"default"`
-	} `json:"labels"`
-	Assignee struct {
-		Login string `json:"login"`
-	} `json:"assignee"`
-	Assignees []struct {
-		Login string `json:"login"`
-	} `json:"assignees"`
-	Milestone struct {
-		Title string `json:"title"`
-		State string `json:"state"`
-	} `json:"milestone"`
-	CreatedAt   time.Time   `json:"created_at"`
-	UpdatedAt   time.Time   `json:"updated_at"`
-	ClosedAt    interface{} `json:"closed_at"`
-	PullRequest struct {
-		URL string `json:"url"`
-	} `json:"pull_request"`
-}
-
-var gIssues []GitIssues
+func getIssuesForRepo(owner string, repo string) (issues []GitIssue, err error) {
+	issuesGH, _, err := buzzClient.Issues.ListByRepo(ctx, owner, repo, nil)
+	if err != nil {
+		return nil, err
+	}
 
-func populateIssues(url string) {
-	pullURL := url + token
-	fmt.Println("Fetching from....", pullURL)
-	resp, err := http.Get(pullURL)
-	exitOnErr(err)
+	// iterate through each issue and scrape what buzz needs.
+	for _, elem := range issuesGH {
+		// var declaration automatically initialize flag var to 0
+		issue := GitIssue{}
 
-	defer resp.Body.Close()
-	htmlData, err := ioutil.ReadAll(resp.Body)
-	exitOnErr(err)
+		if elem.PullRequestLinks != nil {
+			continue
+		}
 
-	mIssues := []RepoIssues{}
-	json.Unmarshal(htmlData, &mIssues)
+		issue.Number = *elem.Number
+		issue.Title = *elem.Title
+		issue.Repo = owner + "/" + repo
```

Comment:
Brandon: 
Could you please test/retest if the issue info shows up in a new tag to when the issue is clicked on, just to make sure?
Thanks.